compiler/test/dotc/pos-test-pickling.blacklist

```diff
@@ -123,3 +123,14 @@ i19955a.scala
 i19955b.scala
 i20053b.scala
 
+# alias types at different levels of dereferencing
```

Comment:
Is this still needed with the latest version? Shouldn't these pass pickling tests?

Reply:
I just checked: they are still needed.